cmd/fs-v1-multipart_test.go

```diff
@@ -32,6 +32,7 @@ import (
 
 // Tests cleanup multipart uploads for filesystem backend.
 func TestFSCleanupMultipartUploadsInRoutine(t *testing.T) {
+	t.Skip()
```

Comment:
No, I will not @klauspost this will be skipped forever - they are not needed anymore as we will not be doing any more feature work on FS mode - eventually removed.

Since we do not support fresh FS deployments it is not easy to write tests with fresh deployments.